aptos-move/framework/README.md

```diff
@@ -47,3 +47,14 @@ The overall structure of the Aptos Framework is as follows:
 ├── releases                                    # Move release bundles
 └── tests
 ```
+
+## Move unit test selection
+In this package, unit test run triggered by `cargo test <filter>` will only select rust test wrappers
+(for example, those marked as `#[test]` in `tests/move_unit_test.rs`).
+
+To select the actual test cases in Move, use the following command.
+```bash
+MOVE_TEST_FILTER=<move-test-filter> cargo test <the-rest-params>
+```
```

Comment:
We should stay away from using environment variables to control configuration, and instead add a flag `--filter` to filter tests

Reply:
+1. I appreciate the attempt here but this is what aptos move test -f filter is designed for

Reply:
I see. Can’t you do cargo run -p aptos — move test …?

Reply:
That will take a really long time to recompile the CLI...

Reply:
use path/to/aptos-core/target/debug/aptos move test

Reply:
How does that address the long time to recompile after modifying a Rust native?

Reply:
Ahhh... fwiw, you can filter by the framework package.

I do like this direction, just not entirely sure how easy it is to get the flags from test and route them appropriately :grimacing: